CorsixTH/Lua/graphics.lua

```diff
@@ -258,11 +258,12 @@ function Graphics:loadRaw(name, width, height, dir, paldir, pal)
   end
   bitmap:setPalette(palette)
   assert(bitmap:load(data, width, self.target))
-  local function reloader(bitmap)
-    bitmap:setPalette(palette)
-    local data = self.app:readDataFile(dir, name .. ".dat")
-    data = data:sub(1, width * height)
-    assert(bitmap:load(data, width, self.target))
+
+  local function reloader(bm)
```

Comment:
Since you renamed the other (below) `reloader` to `font_reloader`, perhaps this could become `bitmap_reloader`?